pastas/stats/tests.py

```diff
@@ -86,7 +90,7 @@ def durbin_watson(series=None):
     return dw_stat, p
 
 
-def ljung_box(series=None, lags=15, nparam=0, full_output=False):
+def ljung_box(series: Series = None, lags: int = 15, nparam: int = 0, full_output: bool = False) -> Tuple[float, float]:
```

Comment:
Why series=None?

Reply:
See comment above for tests.durbin_watson()